common-tools/clas-reco/src/main/java/org/jlab/clas/reco/io/EvioHipoEvent4.java

```diff
@@ -601,7 +603,7 @@ public void fillHipoEventTrueInfo(Event hipoEvent, EvioDataEvent evioEvent){
                         hipoBank.putFloat("avgT",    irow, (float) evioBank.getDouble("avgT",i));
                         hipoBank.putInt("nsteps",    irow,         evioBank.getInt("nsteps",i));
                         hipoBank.putInt("procID",    irow,         evioBank.getInt("procID",i));
-                        hipoBank.putInt("hitn",      irow,         evioBank.getInt("hitn",i));
+                        hipoBank.putInt("hitn",      irow,         evioBank.getInt("hitn",i)+offset);
```

Comment:
This "+offset" should be clarified.

Reply:
I guess @raffaelladevita is added that part.
Maybe she can comment. I think it is related to the fact, that earlier, during the evio2hipo, "hitn" was starting from 0 for both PCal and EC, while they had the same detector.

Reply:
Yes, the offset was due to the fact that in gemc EC and PCAL were separated detectors while in coatjava they are the same